scripts/src/java/org/oppia/android/scripts/proto/copyright_license.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+// Stores the details of a copyright license.
+message CopyrightLicense {
```

Comment:
This doesn't ever seem to be stored on disk. Why not use a data class instead of a proto?

Reply:
+1 dont store it as a proto if it isn't necessary

Reply:
Done, removed.